src/Requests/Transfers24.php

```diff
@@ -161,6 +162,11 @@ class Transfers24
      * @var int|null
      */
     protected $article_number = null;
+    
+    /**
+     * @var string
+     */
+    protected $encoding = Encoding::ISO;
```

Comment:
I believe we shouldn't set default encoding to ISO. I believe we shouldn't set any encoding because it might affect applications using this package already

Reply:
Przelewy24 uses ISO-8859-2 as default encoding

Reply:
Maybe, but I haven't seen this in documentation and UTF-8 seems to be much more reasonable these days.